cmd/manager/main.go

```diff
@@ -82,7 +88,7 @@ func initActuator(m manager.Manager) {
 	}
 
 	params := machineactuator.ActuatorParams{
-		Client:           m.GetClient(),
```

Comment:
Why are we moving from the manager runtime client?

Reply:
Temporary solution until I figure out how to use fake runtime controller. Currently, there is none in upstream [1].

[1] https://github.com/kubernetes-sigs/controller-runtime/tree/master/pkg/manager